sections/header.liquid

```diff
@@ -370,15 +374,16 @@
     {%- endif -%}
         <a href="{{ routes.root_url }}" class="header__heading-link link link--text focus-inset">
           {%- if section.settings.logo != blank -%}
-            {%- assign image_size_2x = section.settings.logo_width | times: 2 | at_most: 5760 -%}
-            <img srcset="{{ section.settings.logo | image_url: width: section.settings.logo_width }} 1x, {{ section.settings.logo | image_url: width: image_size_2x }} 2x"
-              src="{{ section.settings.logo | image_url: width: section.settings.logo_width }}"
-              loading="lazy"
-              class="header__heading-logo"
-              width="{{ section.settings.logo.width }}"
-              height="{{ section.settings.logo.height }}"
-              alt="{{ section.settings.logo.alt | default: shop.name | escape }}"
-            >
+            {%- assign logo_alt = section.settings.logo.alt | default: shop.name | escape -%}
+            {% assign logo_height =  section.settings.logo_width | divided_by: section.settings.logo.aspect_ratio %}
+            {{ section.settings.logo | image_url: width: section.settings.width | image_tag: 
+              loading: 'lazy', 
```

Comment:
I'm wondering if the logo should actually be lazy loaded ? 🤔  since we know it's above the fold, should we prioritize it's loading ?

Reply:
What about “eager” loading since it is above the fold?

Reply:
Removing it since we dont need to lazy load it at the top 🤔

Reply:
@ludoboludo and @sofiamatulis - I researched Google’s outlook for loading above the fold, including eager load. In this some ifuc instance, I would suggest removing the loading function all together. It will eager load since it stays above the fold in the main header. 

**Reference:**
(https://web.dev/browser-level-image-lazy-loading/#avoid-lazy-loading-images-that-are-in-the-first-visible-viewport).

> It is recommended to only add loading=lazy to images which are positioned below the fold, if possible.

> Generally, any images within the viewport should be loaded eagerly using the browser's defaults. You do not need to specify loading=eager for this to be the case for in-viewport images.

Reply:
@sofiamatulis - sorry I missed your comment above mine. We are on the same page. ❤️

Reply:
Great! Thanks for sharing the resource @bacounts 🤗